tests/runtime/out_stackdriver.c

```diff
@@ -1427,8 +1557,11 @@ TEST_LIST = {
     {"operation_subfields_in_incorrect_type", flb_test_operation_incorrect_type_subfields},
     {"operation_extra_subfields_exist", flb_test_operation_extra_subfields},
     {"resource_k8s_container_common", flb_test_resource_k8s_container_common },
+    {"resource_k8s_container_no_local_resource_id", flb_test_resource_k8s_container_no_local_resource_id },
     {"resource_k8s_node_common", flb_test_resource_k8s_node_common },
+    {"resource_k8s_node_no_local_resource_id", flb_test_resource_k8s_node_no_local_resource_id },
     {"resource_k8s_pod_common", flb_test_resource_k8s_pod_common },
+    {"resource_k8s_pod_no_local_resource_id", flb_test_resource_k8s_pod_no_local_resource_id },
```

Comment:
Do you have a test case when tag and local_resource_id are all empty?
I think the test cases should be 
1. success (read from local_resource_id)
2. success (read from tag)
3. failure (failed to process local_resource_id)

Reply:
Hi Jie, thanks for the review. I think tag will never be empty as tag is mandatory for the log event. For the test number 3. failure (failed to process local_resource_id), I don't think the current testing framework supports tests that will throw out the error... 
@edsiper Hi Eduardo, do you have any idea/advice on testing the case that the program will throw the error during the runtime?